.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+- '2.7'
+install: pip install -r requirements.txt --use-mirrors
+script: nosetests
+deploy:
```

Comment:
We may want to add `distributions: "sdist bdist_wheel"` because otherwise only a `sdist` is uploaded to PyPI. Wheels is apparently a [newer package standard](https://www.python.org/dev/peps/pep-0427/) we'll want to support.

Travis claims to handle installing the `wheels` dependency for us when `bdist_wheel` is specified.